modular_zubbers/modules/customization/modules/mob/living/carbon/human/species/tajaran.dm

```diff
@@ -0,0 +1,98 @@
+//Tajaran Bubber edit
+//makes them more like Citrp's tajara aka snow cats
+
+/datum/species/tajaran
+	inherent_traits = list(
+		TRAIT_ADVANCEDTOOLUSER,
+		TRAIT_CAN_STRIP,
+		TRAIT_LITERATE,
+		TRAIT_HATED_BY_DOGS,
+		TRAIT_MUTANT_COLORS,
+		TRAIT_CATLIKE_GRACE,
+	)
+	mutanteyes = /obj/item/organ/internal/eyes/tajaran
+	mutantears = /obj/item/organ/internal/ears/cat/tajaran
+	//Cold resistance
+	coldmod = 0.77
+	heatmod = 1.15
+	bodytemp_normal = BODYTEMP_NORMAL + 5 //Even more cold resistant, even more flammable
+	bodytemp_heat_damage_limit = (BODYTEMP_HEAT_DAMAGE_LIMIT - 20)
+	bodytemp_cold_damage_limit = (BODYTEMP_COLD_DAMAGE_LIMIT - 20)
+	meat = /obj/item/food/meat/slab/human/mutant/feline //you monster!
+	skinned_type = /obj/item/stack/sheet/animalhide/cat
+	ass_image = 'modular_zubbers/icons/ass/asstajara.png'
+
+//Tajaran tongue
+/obj/item/organ/internal/tongue/cat/tajaran
+	name = "tajaran tongue"
+	modifies_speech = TRUE
+	languages_native = list(/datum/language/siiktajr)
+
+/obj/item/organ/internal/tongue/cat/tajaran/modify_speech(datum/source, list/speech_args)
+	var/static/regex/tajara_roll = new("r+", "g")
+	var/static/regex/tajara_roLL = new("R+", "g")
+	var/message = speech_args[SPEECH_MESSAGE]
+	if(message[1] != "*")
+		message = tajara_roll.Replace(message, "rrr")
+		message = tajara_roLL.Replace(message, "RRR")
+		message = replacetext(message, "r", "r")
+//Insert russion translations here (sorry russions)
+	speech_args[SPEECH_MESSAGE] = message
+
+//Tajara have the innate ability to see in the dark better than most
+/obj/item/organ/internal/eyes/tajaran
+	name = "tajaran eyes"
+	desc = "they seem very cat like."
+	flash_protect = FLASH_PROTECTION_HYPER_SENSITIVE //sorry cat gamers it's over
+	color_cutoffs = list(12, 7, 7)
```

Comment:
Bugs found working on a hot fix for them the heterochromatic trait is non functional

Reply:
weird this issue seems to be only for places that don't load your character prefs from start

Reply:
Unrelated issue, out of scope. I should finally look into what may be causing it